doc/source/release-notes/Percona-Server-5.5.51-38.1.rst

```diff
@@ -0,0 +1,62 @@
+.. rn:: 5.5.51-38.1
+
+==============================
+ |Percona Server| 5.5.51-38.1
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.51-38.1 on
+August 19th, 2016. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.51-38.1/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.51
+<http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-51.html>`_, including
+all the bug fixes in it, |Percona Server| 5.5.51-38.1 is now the current stable
+release in the 5.5 series. All of |Percona|'s software is open-source and free,
+all the details of the release can be found in the `5.5.51-38.1 milestone at
+Launchpad <https://launchpad.net/percona-server/+milestone/5.5.51-38.1>`_.
+
+Bugs Fixed
+==========
+
+ :ref:`pam_plugin` would abort authentication if there were more than a
+ thousand members in the group. Bug fixed :bug:`1608902`.
+
+ If ``DROP DATABASE`` would fails to delete any of the tables, command is still
```

Comment:
If DROP DATABASE would fail to delete some of the tables in the database, the partially-executed command is logged in the binlog as "DROP TABLE t1, t2, ... " for the tables for which drop succeeded. A slave might fail to replicate such DROP TABLE statement if there exist foreign key relationships to any of the dropped tables and the slave has a different schema from master. Fix by checking, on the master, whether any of the database to be dropped tables participate in a FK relationship, and fail the DROP DATABASE statement immediately. Bug fixed ...